__tests__/pages/login.test.js

```diff
@@ -171,4 +173,39 @@ describe('Login Page', () => {
 
     await waitFor(() => expect(signIn).toBeCalled())
   })
+
+  test('Should show AlreadyLoggedIn component if there is a session', async () => {
```

Comment:
Asynchronous tests need `expect.assertions(x)` to make sure they actually run. In this case, I would put `expect.assertions(1)` at the top as I'm expecting one assertion to be run (`expect(getByText('You are already logged in.')).toBeTruthy()`)

Reply:
I updated the test files to have expect.assertions(1) at the top of each test.